core/storage/suggestion/gae_models.py

```diff
@@ -610,3 +613,83 @@ def export_data(cls, user_id):
                 'rejection_message': voiceover_model.rejection_message
             }
         return user_data
+
+
+class ReviewerAndSuggestionCountsModel(base_models.BaseModel):
+    """Records the total number of reviewers for each suggestion type and the
+    total number of suggestions in review for each suggestion type. There is
+    only ever one instance of this model with ID
+    REVIEWER_AND_SUGGESTION_COUNTS_MODEL_ID.
+    """
+
+    # A dictionary where the keys represent the languages that translation
+    # suggestions are offered in and the values correspond to the total number
+    # of reviewers who have permission to review translation suggestions in
+    # that language.
+    translation_reviewer_counts_by_lang = ndb.JsonProperty(
+        required=True, indexed=True)
+    # A dictionary where the keys represent the languages that translation
+    # suggestions are offered in and the values correspond to the total number
+    # of translation suggestions that are currently in review in that language.
+    translation_suggestion_counts_by_lang = ndb.JsonProperty(
+        required=True, indexed=True)
+    # The total number of reviewers who have permission to review question
+    # suggestions.
+    question_reviewer_count = ndb.IntegerProperty(
+        required=True, indexed=True)
+    # The total number of question suggestions that are currently in review.
+    question_suggestion_count = ndb.IntegerProperty(
+        required=True, indexed=True)
+
+    @classmethod
+    def get(cls, strict=True):
+        """Gets the ReviewerAndSuggestionCountsModel instance. If the
+        ReviewerAndSuggestionCountsModel does not exist yet, it is created.
+        This method helps enforce that there should only ever be one instance
+        of this model.
+
+        Returns:
+            ReviewerAndSuggestionCountsModel. The
+            ReviewerAndSuggestionCountsModel.
```

Comment:
```suggestion
            ReviewerAndSuggestionCountsModel. The single model instance.
```

Slightly more specific on exactly what is being returned.

Reply:
Changed.